index.js

```diff
@@ -0,0 +1,15 @@
+function ConvertToHTML(element) {
```

Comment:
what's the point of this function if u r using mitril to render elements?

Reply:
It would be more clearer with the next PR where I would have another function calling this function, when there is nested DOM elements.
Such as: Experience Description has a URL link, so it would look like this in mithril.js
m(div, [text, m(a, {href}, url)])